### PR TITLE
[API] Fix top_k_top_p_sampling_from_logits param typo

### LINKED ISSUE #873: [API] Typo in parameter list for fused sampling kernel (top_k_top_p_sampling_from_logits)
There appears to be a param name mistake in the API [for sampling.top_k_top_p_sampling_from_logits](https://github.com/flashinfer-ai/flashinfer/blob/78dde79bffd51cb862bd2c553bb4a3756902ebd4/flashinfer/sampling.py#L706). It should be logits instead of probs here right? That's what the docs below say too at least.

### PR DESCRIPTION
**Pull request by kasohrab**
Resolves #873

### CODE REVIEW

**Review by yzh119 — APPROVED** [review]
LGTM, thanks for the fix.
